app/Http/Controllers/Lti3Controller.php

```diff
@@ -53,17 +53,16 @@ public function index(Request $request)
      * @throws LtiException
      */
     public function launch(Request $request)
-    {
+    {   
         $config_directory = $request->query("config_directory", "configs");
         logger("LTI3Controller config directory:". $config_directory);
         $launch = LTI\LTI_Message_Launch::new(new LTI3_Database($config_directory));
         try {
-            $launch->validate();
+            $launch->validate(true);
```

Comment:
Php linteren min reagerte på validate(true) når jeg går til definisjon: LTI_Message_launch.php  er den definert som: public function validate(array $request = null). Jeg antar dette betyr at den tar verdien null eller array

Reply:
Nye LTI_Message_Launch.php her: https://github.com/matematikk-mooc/lti-1-3-php-library/blob/master/src/lti/LTI_Message_Launch.php (tidlgere tok den ikke inn noen parametere), tar inn boolean verdi for å skippe validering av state, da state nå valideres i NoCookies.vue